site/en/docs/webstore/account-deletion/index.md

```diff
@@ -15,13 +15,22 @@ things you need to understand before you delete an account:
 
 - Account deletion is permanent. There is no "undo" for this action.
 - You can't delete an account that still has published items. If you have published items, make sure
-  to [unpublish][unpublish] them before you try and delete the account.
+  to [unpublish](#remove-listing) them before you try and delete the account.
 - Group publisher accounts can only be deleted by the owner or manager of the associated Google
   Group.
 
+### How to remove my listing from the store? {: #remove-listing }
+
+Visit the [developer dashboard][dev-dashboard]. Click the item you want to
```

Comment:
This should probably be steps, but I don't know if we have time.

Reply:
I'll do it on a separate PR on Monday